File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/metaschema/FaseFenologica.ts
import {BaseCodeDescr} from '../baseClass/baseCodeDescr';
import {Specie} from "./utilizzi/Specie";

export class FaseFenologica extends BaseCodeDescr {
    stadioCrescitaBBCH: BaseCodeDescr;
    specieVegetale: Specie;
    stadio: string;
    fioritura: boolean;

    constructor(codice: number,descrizione: string) {
      super(codice,descrizione);
    }
}

File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreModelsSTD/metaschema/FaseFenologica.cs
using AgronicaCoreModelsSTD.baseClass;
using AgronicaCoreModelsSTD.metaschema.utilizzi;

namespace AgronicaCoreModelsSTD.metaschema
{
    public class FaseFenologica: BaseCodeDescr
    {
        public FaseFenologica() : base() { }

        public FaseFenologica(int codice) : base(codice, "") { }

        public BaseCodeDescr stadioCrescitaBBCH { get; set; }

        public Specie specieVegetale { get; set; }

        public string stadio { get; set; }

        public bool fioritura { get; set; }
    }
}
